### ISSUE TITLE
Cannot create schema: no namespace for types found

### ISSUE THREAD

**Issue by smilesrg** [thecodingmachine/graphqlite#541]
I've got an issue with Symfony 5.4 and `thecodingmachine/graphqlite:"5.0.2"`
```
In SchemaFactory.php line 395:
                                                                               
  Cannot create schema: no namespace for types found (You must call the SchemaFactory::addTypeNamespace() at least once). 
```

**Comment by pvlg** [comment]
I also encountered such a problem. The reason turned out to be the latest version of symfony/dependency-injection. Downgraded to a previous version

**Comment by oojacoboo** [comment]
Are you using the Symfony GraphQLite bundle?  I suspect the issue is related to that and not GraphQLite.

**Comment by smilesrg** [comment]
Yes, I'm using `"thecodingmachine/graphqlite-bundle": "^5.4"` with all updated depencencies.

**Comment by oojacoboo** [comment]
You'll want to direct any issues with the bundle and how it's handling autoloading to that repo, specifically.
